.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
This isn't ignored on purpose. If you run into issues with it please explain on Slack.